doc/sphinx/source/using.rst

```diff
@@ -407,6 +407,29 @@ pocl.
  When set to 1, prints out remarks produced by the loop vectorizer of LLVM
  during kernel compilation.
 
+- **POCL_FORCE_VECTOR_WIDTH**
+
+ Forces the LLVM loop vectorizer to use the specified vector width, overriding
```

Comment:
Is the "vector width" measured in number of elements or bytes (since the other env variable measures in bits)? Perhaps this should be clarified.

Reply:
It's a number of loop iterations.  I've tried to clarify that here: ff3299db3391a3aaa9327ed5f5c485eee6667f00